packages/eds-core-react/src/components/Popover/PopoverContent.tsx

```diff
@@ -3,7 +3,6 @@ import styled, { css } from 'styled-components'
 
 const ContentWrapper = styled.div(({ theme }) => {
   return css`
-    overflow: hidden;
```

Comment:
I would revert this and instead fix it in #2517. Need to make sure popover works with a lot of content that fills the height (I assume header and actions should be fixed while content scrolls)